tasks/main.yml

```diff
@@ -45,3 +45,10 @@
 
 - name: Set local timezone to UTC
   file: src=/usr/share/zoneinfo/Etc/UTC dest=/etc/localtime state=link force=true
+
+- name: Schedule cron job to track security updates of held packages
+  cron:
+    name: "security updates checker"
+    minute: "0"
+    hour: "1"
+    job: security_updates_checker.sh
```

Comment:
The job option in the cron module in Ansible takes a command - it won't automatically upload a file or anything like that. There needs to be a separate step to upload the file somewhere sane on the system (like /usr/local/sbin) with proper permissions. For example, see the job in the [backup play|https://github.com/open-craft/ansible-backup-to-tarsnap/blob/75c64e2a13b12d7555e40edb7d178c42b518b7ff/tasks/main.yml#L27-L44] (except in this case, you can just use copy the file rather than templating it).